File: Examples/TS/test2.ts
function concat(l1: number[], l2: number[]): number[] {
    if (l1.length === 0) {
        return l2;
    } else {
        const h= l1[0];
        const t= l1.slice(1);
        return [h, ...concat(t, l2)];
    }
}
const x = [5, 4]
const y = [8, 9, 10]
const r = concat(x, y)

print(r)

File: Examples/PY/test2.py
def concat(l1: list[int], l2: list[int]) -> list[int]:
    match l1:
        case [h, *t]:
            return [h, *(concat(t, l2))]
        case _:
            return l2
x = [5, 4]
y = [8, 9, 10]
r = concat(x, y)

print(r)
